dockers/app/Dockerfile

```diff
@@ -1,27 +1,55 @@
-FROM isard/alpine-pandas:latest
+FROM alpine:3.8 as production
 MAINTAINER isard <info@isard.com>
 
-RUN apk add --no-cache bash yarn py3-libvirt py3-paramiko py3-lxml py3-pexpect py3-openssl py3-bcrypt py3-gevent py3-flask py3-netaddr py3-requests curl openssh-client
+RUN apk add --no-cache \
+	yarn \
+	py3-libvirt \
+	py3-paramiko \
+	py3-lxml \
+	py3-pexpect \
+	py3-openssl \
+	py3-bcrypt \
+	py3-gevent \
+	py3-flask \
+	py3-netaddr \
+	py3-requests \
+	curl \
+	openssh-client \
+	supervisor
+
+RUN apk add --no-cache --virtual .build_deps \
+	build-base \
+	python3-dev \
+	libffi-dev \
+	openssl-dev 
+RUN pip3 install pandas
+RUN apk del .build_deps
 
 COPY dockers/app/requirements.pip3 /requirements.pip3
 RUN pip3 install --no-cache-dir -r requirements.pip3
 
-RUN mkdir -p /root/.ssh
-RUN echo "Host isard-hypervisor \
+# Create the required directories
+RUN mkdir -p /var/log/supervisor /isard /root/.ssh
+
+# Configure SSH
+RUN echo -e "Host isard-hypervisor\n \
 	StrictHostKeyChecking no" >/root/.ssh/config
 RUN chmod 600 /root/.ssh/config
 
-RUN apk add --no-cache supervisor
-RUN mkdir -p /var/log/supervisor
+# Copy the isard source
+COPY ./src /isard
+RUN mv /isard/isard.conf.docker /isard/isard.conf
+
+COPY dockers/app/certs.sh /
+COPY dockers/app/add-hypervisor.sh /
 COPY dockers/app/supervisord.conf /etc/supervisord.conf
 
 EXPOSE 5000
 
-COPY dockers/app/certs.sh /
-COPY dockers/app/add-hypervisor.sh /
+CMD ["/usr/bin/supervisord", "-c", "/etc/supervisord.conf"]
 
-RUN mkdir /isard
-ADD ./src /isard
-RUN mv /isard/isard.conf.docker /isard/isard.conf
+FROM production as development
+RUN pip3 install ipython pytest
```

Comment:
It should be `pip3 install --no-cache-dir`

Reply:
fixed